cgi-bin/ljlib.pl

```diff
@@ -39,6 +39,7 @@ BEGIN
 
 BEGIN {
     # mod_perl does this early too, make sure we do as well
+    $LJ::_T_RUNNING = 1 if $0 && $0 =~ m!^t/!;
```

Comment:
Do you prefer this over having something in each test file that says "I am a test, you are in the test environment"?